test/API/README.md

```diff
@@ -44,16 +45,19 @@ These API tests currently only support usage with the native HDF5 VOL connector
 connectors that can be loaded dynamically as a plugin. For information on how to build a VOL
 connector in this manner, refer to section 2.3 of the [HDF5 VOL Connector Author Guide](https://hdfgroup.github.io/hdf5/develop/_v_o_l__connector.html).
 
-TODO: section on building VOL connectors alongside HDF5 for use with tests
+If an HDF5 VOL connector has been built alongside the library, these API tests can be used
+for testing that VOL connector at the same time that HDF5 is test. For more information on
+how to build a VOL connector alongside the HDF5 library, refer to
+[Building and testing HDF5 VOL connectors with CMake FetchContent](https://hdfgroup.github.io/hdf5/develop/df/d6b/_c_make_vols.html).
```

Comment:
@byrnHDF This is the correct link, but the `df/d6b` part looks suspicious with regards to how permanent this link is. Is there a more appropriate form?

Reply:
just created Change md file links to correct locations #5349 to deal with all md links. Yes it should be permanent, but it is at the whim of doxygen.